pynicotine/gtkgui/ui/popovers/searchfilters.ui

```diff
@@ -108,7 +117,7 @@
                         <property name="xalign">0</property>
                         <property name="margin-start">12</property>
                         <property name="label" translatable="yes">For order-insensitive filtering, as well as filtering several exact phrases, vertical bars can be used to separate phrases and words.
-    Example: Spears|Brittany|My beautiful album|hello</property>
+    Example: Twin|Aphex|Analogue Bubblebath|Digeridoo</property>
```

Comment:
As you mentioned, this should be replaced with a non-copyrighted example before the PR is merged.